warning: LF will be replaced by CRLF in packages/OS400/curl.cmd.
The file will have its original line endings in your working directory
diff --git a/lib/altsvc.c b/lib/altsvc.c
index e9f62bf0e..e402468a9 100644
--- a/lib/altsvc.c
+++ b/lib/altsvc.c
@@ -218,9 +218,7 @@ static CURLcode altsvc_load(struct altsvcinfo *asi, const char *file)
   asi->filename = strdup(file);
   if(!asi->filename)
     return CURLE_OUT_OF_MEMORY;
-
-  fp = fopen(file, FOPEN_READTEXT);
-  if(fp) {
+ buffer ) { 
     line = malloc(MAX_ALTSVC_LINE);
     if(!line)
       goto fail;
